octoprint_slicer/static/js/slicer.js

```diff
@@ -499,6 +506,42 @@ $(function() {
             self.renderer.render( self.scene, self.camera );
         };
 
+	self.slicerProperties = ko.observable();
+	self.updateSlicerProperties = function(newSlicersArray) {
+	    if (_.isArray(newSlicersArray) &&
+		newSlicersArray.length > 0) {
+		OctoPrint
+		    .get(OctoPrint.getBlueprintUrl("slicer") +
+			 "slicer/properties")
+		    .then(function (result) {
+			self.slicerProperties(result);
+		    });
+	    } else {
+		self.slicerProperties({});
+	    }
+	}
+	self.slicingViewModel.slicers.subscribe(self.updateSlicerProperties);
+
+	self.sameDevice = ko.computed(function() {
```

Comment:
Just a lot of grep to find the same_device.

For most people, I get that same_device is True, like you said.  So I'll modify this PR so that we just assume same_device is True for all users.  If @foosel later accepts https://github.com/foosel/OctoPrint/issues/1748 , we can add the support for same_device.  That'll be easy.

I'll start working on the new PR.